src/main/resources/application.properties

```diff
@@ -2,9 +2,10 @@ server.port=8080
 server.servlet.context-path=/banter-board
 initialize-test-data=true
 
-spring.data.mongodb.host=localhost
+# To run locally change the below line to localhost
+spring.data.mongodb.host=${MONGO_HOST}
```

Comment:
I don't think we should do this. This makes the Spring Boot app need the environment variable MONGO_HOST defined or else it won't work. Try running the app just from Intellij, for example.